src/components/FrontPageButton.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+import Teacher from './TeacherPage';
+import style from './buttons.scss';
+
+const FrontPageButton = React.createClass({
+
+  getInitialState() {
+    return {
+      info: null
+    }
+  },
+
+  displayTeacherContent(info) {
+    this.setState({info});
+  },
+
+  navigate() {
+    this.props.history.pushState(null, '/scratch/flagg/README');
```

Comment:
Her tror jeg det er bedre å bruke ` this.context.router.push` (se f.eks. hvordan det er gjort i CourseItem.js).

Tror forøvrig ikke router er satt opp til å støtte README, så kanskje bedre å ha '/scratch/flagg/flagg' for nå?